src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -672,36 +678,80 @@ private static string GetCommonSuffix(string firstPath, string secondPath)
         private void AttachToDocumentChanges(uint docCookie, IVsWindowFrame pFrame)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
+
+            if (RunIndexToRunDataCache == null)
```

Comment:
>== null [](start = 39, length = 7)

is null would be better here.

Reply:
There are "== null" checks everywhere, leaving for consistency

---
In reply to: [489583167](https://github.com/microsoft/sarif-visualstudio-extension/pull/195#discussion_r489583167) [](ancestors = 489583167)